hack/latest-kind-images.sh

```diff
@@ -17,23 +17,20 @@
 
 set -eu -o pipefail
 
-# source this lib for KIND_IMAGE_REPO
-source "devel/lib/lib.sh"
+export KIND_IMAGE_REPO="docker.io/kindest/node"
 
 CRANE=crane
 TAGS=$(mktemp)
 
 trap 'rm -f -- "$TAGS"' EXIT
 
-if ! which $CRANE >/dev/null 2>&1; then
+if ! command -v $CRANE >/dev/null 2>&1; then
 	echo -e "Couldn't find crane. Try running:\ngo install github.com/google/go-containerregistry/cmd/crane@latest" >&2
 	exit 1
 fi
 
 function latest_kind_tag () {
-	# TODO (irbekrm): fix this so it sorts semver tags properly. 'sort'
-	# seems to consider vX.Y.2 bigger than vX.Y.12
-	grep -E "^v$1" $TAGS | sort | tail -1
+	grep -E "^v$1" $TAGS | sort --version-sort | tail -1
```

Comment:
note: thanks @irbekrm for spotting this! and thanks `sort` for having this functionality built in

I checked via @jakexks that macOS's `sort` also has `--version-sort` :apple: